src/css/main.scss

```diff
@@ -2,7 +2,7 @@
   --sn-stylekit-base-font-size: 13px;
 
   --sn-stylekit-font-size-p: 1.0rem;
-  --sn-stylekit-font-size-editor: 1.21rem;
+  --sn-stylekit-font-size-editor: 1.23rem;
```

Comment:
I was proposing that we make `--sn-stylekit-font-size-editor` to be about `16px` because `16px` seems to be the standard font size for paragraph text. I was thinking about using `1.23rem` because it's `15.99px`, but now I wonder, is there a reason to prefer `1.23rem` over using `16px` directly? Perhaps we could use `16px` directly instead of using `rem`?

If we use `rem`, then I suppose `1.231rem` might be closer to `16px` than `1.23rem` (`16.003px` vs `15.99px`, so maybe we could use that instead?

Reply:
I always thought our editor font was too large tbh. So not sure about making it bigger. Other note taking apps use even smaller sizes, like 12 or 14px.

Reply:
I checked MDN and they recommend against px: https://developer.mozilla.org/en-US/docs/Web/CSS/font-size#pixels

I guess it's pretty hard to come to a consensus about what the default font size should be since we have different preferences 🤔 I always found it challenging to use the official editors because the font size was too small for me 😅

Reply:
Since font options is something we plan to provide at the app level I've updated the PR back to status quo